ngraph/python/tests/test_onnx/test_ops_matmul.py

```diff
@@ -18,7 +18,10 @@
 from onnx.helper import make_graph, make_model, make_node, make_tensor_value_info
 
 from tests.runtime import get_runtime
-from tests.test_onnx.utils import import_onnx_model
+from tests.test_onnx.utils import import_onnx_model, xfail_issue_35916, xfail_issue_35917, \
+    xfail_issue_35918, xfail_issue_35921
```

Comment:
[SUGESTION]
Importing xfail_issues in the new line is more readable, isn't it?
```suggestion
from tests.test_onnx.utils import import_onnx_model, \
    xfail_issue_35916, \
    xfail_issue_35917, \
    xfail_issue_35918, \
    xfail_issue_35921
```

Reply:
Thanks for suggestion, It is something with this approach. But anyway, we have a lot of tests files and one file with list of all xfail_issues. I think that one list is sufficient to understand how many issues we have. Also the names are so intuitive and all affected definition is wrapped also.

Reply:
If I'm not mistaken following syntax is also correct:
```
from tests.test_onnx.utils import (
  import_onnx_model,
  xfail_issue_35916,
 # and so on..
)

Reply:
> If I'm not mistaken following syntax is also correct:
> 
> ```
> from tests.test_onnx.utils import (
>   import_onnx_model,
>   xfail_issue_35916,
>  # and so on..
> )
> ```

It doesn't work in Python. I propose then the approach:
import (lib1,
             lib2,
             ...,
             )